web/styles/zulip.css

```diff
@@ -2699,6 +2699,10 @@ select.invite-as {
     }
 }
 
+.fa-copy:hover::before {
```

Comment:
I think we should move this CSS above in the "#about-zulip" selector.

I don't think we use `fa-copy` at any other place currently, but still would be good to have the CSS properly scoped in case we do not want this behavior when we add more copy icons in future. And if we want we can anyway extract it out to use as common CSS in future.

Reply:
What do we use for "Copy and close" and "Copy code" in the message feed?

Reply:
Also the copy button for the email in the user card, if it's too long to fit.

Reply:
Anyway, those are the places that come to mind where we currently have a "copy" button along these lines.

Reply:
Looks like fa-copy is referenced [here](https://github.com/zulip/zulip/blob/18d8fbc74be15ac20ee74433abe3263d8e8b7208/templates/analytics/realm_details.html#LL11C1-L11C1) and [here](https://github.com/zulip/zulip/blob/18d8fbc74be15ac20ee74433abe3263d8e8b7208/templates/analytics/support.html#L93). @sahil839

Reply:
> Looks like fa-copy is referenced [here](https://github.com/zulip/zulip/blob/18d8fbc74be15ac20ee74433abe3263d8e8b7208/templates/analytics/realm_details.html#LL11C1-L11C1) and [here](https://github.com/zulip/zulip/blob/18d8fbc74be15ac20ee74433abe3263d8e8b7208/templates/analytics/support.html#L93). @sahil839

All of those `fa-copy` elements are inside `a` tag and thus already have bootstrap CSS to handle the hover behavior. Additionally, I guess CSS in `zulip.css` is not applied to analytics pages. So would be better to move this inside `#about-zulip`.

Reply:
The ones that Alya mentioned anyway use different icons and not `fa-copy` icon.

Reply:
Got it, thanks for clarifying. Will move it inside `#about-zulip`.

Reply:
@sahil839 I have changed the commit message and moved the css into the `#about-zulip` section. Please review it and let me know if everything is all right.